src/admin/shared/components/ContentPicker/ContentPicker.tsx

```diff
@@ -80,91 +67,111 @@ const ContentPicker: FunctionComponent<ContentPickerProps> = ({
 					);
 				});
 		}
-	}, [currentType, t]);
-
-	const onChangeText = (value: string) => {
-		setInput(value);
-		onSelect(parsePickerItem('EXTERNAL_LINK', value));
 	};
 
-	const onChangeType = (selected: ValueType<PickerTypeOption>) => {
+	useEffect(() => {
+		inflatePicker(null, setOptions);
+	}, [currentType]); // eslint-disable-line
+
+	// events
+	const onSelectType = (selected: ValueType<PickerTypeOption>) => {
 		setCurrentType(selected as PickerTypeOption);
-		setCurrentValue(null);
+		setCurrentItem(null);
```

Comment:
als je content kiest, en dan de "over" pagina en dan selecteer je opnieuw content, dan reset hij de over pagina.
Als het type hetzelfde blijft moet je de waarde niet resetten:
```
if (selected !== currentType) {
		setCurrentType(selected as PickerTypeOption);
		setCurrentValue(null);			setCurrentItem(null);
}
```

Reply:
Bedoel je de "Publicatiedetails" pagina? Zie niet direct "over" staan. 😅

Reply:
![image](https://user-images.githubusercontent.com/1710840/75321490-2d226900-5871-11ea-9927-59870fe612ba.png)

als ik in deze screenshot opnieuw "Content" kies in de dropdown, dan verdwijnt de "over" pagina. Je moet enkel de value resetten als je een ander content type kiest dan het huidig geselecteerde content type

Reply:
Ok, bedankt voor de extra uitleg. Nu snap ik wat je bedoelde.